pkg/api/convertertoapi.go

```diff
@@ -102,7 +102,7 @@ func ConvertV20170701ContainerService(v20170701 *v20170701.ContainerService, isU
 }
 
 // ConvertVLabsContainerService converts a vlabs ContainerService to an unversioned ContainerService
-func ConvertVLabsContainerService(vlabs *vlabs.ContainerService, isUpdate bool) *ContainerService {
+func ConvertVLabsContainerService(vlabs *vlabs.ContainerService, isUpdate bool) (*ContainerService, error) {
```

Comment:
Main concern is to make sure that this isn't used by projects that depend on us.  I checked but please confirm :)

Reply:
Yep, that's just us